Robot Code/src/org/usfirst/frc/team3316/robot/commands/DBugCommandGroup.java

```diff
@@ -11,8 +11,8 @@
  */
 public abstract class DBugCommandGroup extends CommandGroup 
 {
-	DBugLogger logger = Robot.logger;
-	Config config = Robot.config;
+	protected DBugLogger logger = Robot.logger;
+	protected Config config = Robot.config;
```

Comment:
You can refer to static variable from a dynamic context without needing to change the syntax. It simply means that there won't be a different reference in each instance of this class. If it really matters I don't mind keeping it this way.